src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -1330,18 +889,19 @@ describe('findBreakingChanges', () => {
   });
 
   it('should detect if a directive was explicitly removed', () => {
-    const oldSchema = new GraphQLSchema({
-      directives: [GraphQLSkipDirective, GraphQLIncludeDirective],
-    });
+    const oldSchema = buildSchema(`
+      directive @DirectiveThatIsRemoved on FIELD_DEFINITION
+      directive @DirectiveThatStays on FIELD_DEFINITION
+    `);
 
-    const newSchema = new GraphQLSchema({
-      directives: [GraphQLSkipDirective],
-    });
+    const newSchema = buildSchema(`
+      directive @DirectiveThatStays on FIELD_DEFINITION
+    `);
 
     expect(findRemovedDirectives(oldSchema, newSchema)).to.eql([
       {
         type: BreakingChangeType.DIRECTIVE_REMOVED,
-        description: `${GraphQLIncludeDirective.name} was removed`,
+        description: `DirectiveThatIsRemoved was removed`,
```

Comment:
Change made for the same motive than above, `printSchema` does not print GraphQL default directives

Reply:
👍